media/js/src/GraphViewer.jsx

```diff
@@ -87,664 +87,109 @@ export default class GraphViewer extends React.Component {
             const parsed = reader.parse(this.props.gInstructions);
             const instructions = writer.render(parsed);
 
-            instructionsEl = <p className="lead text-secondary" dangerouslySetInnerHTML={{__html:instructions}}></p>;
+            instructionsEl = <p className="lead" dangerouslySetInnerHTML={{__html:instructions}}></p>;
         }
 
-        let leftSide = null;
+        const commonViewerProps = {
+            isInstructor: isInstructor,
+            displayLabels: displayLabels,
+            displaySliders: displaySliders,
+            updateGraph: this.updateGraph,
+        };
+
+        let leftSide = (
+            <>
+                <JXGBoard
+                    gType={0}
```

Comment:
This still needs updating here, to not hardcode the gType and re-introduce the loading message.

Reply:
Sorry, forgot it was in this file as well.